jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamInvokeRunner.kt

```diff
@@ -118,32 +117,26 @@ class SamInvokeRunner : AsyncProgramRunner<RunnerSettings>() {
             }.whenComplete { _, exception ->
                 AwsResourceCache.getInstance(state.environment.project)
                     .getResource(StsResources.ACCOUNT, lambdaSettings.region, lambdaSettings.credentials)
-                    .whenComplete { account, _ ->
-                        TelemetryService.getInstance().record(
-                            "SamInvoke",
-                            TelemetryService.MetricEventMetadata(
-                                awsAccount = account ?: METADATA_INVALID,
-                                awsRegion = lambdaSettings.region.id
-                            )
-                        ) {
+                    .whenComplete { _, _ ->
+                        TelemetryService.getInstance().record(state.environment.project) {
```

Comment:
Should this be the Lambda account/region or the IDE account/region

Reply:
true, reverted change